BootloaderCorePkg/Library/SmbiosInitLib/SmbiosTables.h

```diff
@@ -0,0 +1,199 @@
+/** @file
+
+  Copyright (c) 2019, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php.
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#ifndef __SMBIOS_TABLES_H__
+#define __SMBIOS_TABLES_H__
+
+#include <Library/BaseLib.h>
+#include <IndustryStandard/SmBios.h>
+
+#define TO_BE_FILLED                                0
+
+#define SMBIOS_MISC_STRING_NULL                     0
+#define SMBIOS_MISC_STRING_1                        1
+#define SMBIOS_MISC_STRING_2                        2
+#define SMBIOS_MISC_STRING_3                        3
+#define SMBIOS_MISC_STRING_4                        4
+#define SMBIOS_MISC_STRING_5                        5
+#define SMBIOS_MISC_STRING_6                        6
+#define SMBIOS_MISC_STRING_7                        7
+#define SMBIOS_MISC_STRING_8                        8
+#define SMBIOS_MISC_STRING_9                        9
+
+#define SMBIOS_MISC_UNKNOWN                         "UNKNOWN"
+
+///
+/// SMBIOS_TABLE_TYPE00 Default Strings
+///
+///
+#define SMBIOS_MISC_TYPE00_VENDOR                   L"Intel Corporation"
+#define SMBIOS_MISC_TYPE00_BIOS_VERSION             L"CFL_S/CFL_H/WHL_U SBL"
```

Comment:
Should no have platform specific info, could the value comes from VerInfo.txt, e.g. using ImageId + MajorVersion + MinorVersion.